tests/pxScene2d/test_rtHttpRequest.cpp

```diff
@@ -242,39 +242,45 @@ class rtHttpRequestTest : public testing::Test
     ref = req;
 
     rtFileDownloadRequest* dwnl_OK = new rtFileDownloadRequest("https://example.com", req, rtHttpRequest::onDownloadComplete);
-
+    
+    req->AddRef();
```

Comment:
Please remove all changes to test_rtHttpRequest and follow the approach outlined in the previous comment.  If you follow this approach then no changes should be needed in the unit tests.